src/plugins/cal-events/cal-events.js

```diff
@@ -168,15 +168,16 @@ var componentName = "wb-calevt",
 					}
 				]
 			},
-			objEventsList = obj.find( "ol > li, ul > li" ),
-			iLen = objEventsList.length,
+			objEventsList = obj.find( "ol, ul" ).first(),
+			objEventsListItems = objEventsList.find( "> li:not(.wb-fltr-out)" ),
```

Comment:
We will need a working example that illustrate this new functionality of filtering + documentation